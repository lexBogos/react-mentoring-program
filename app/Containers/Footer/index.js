import React from 'react';
import './index.scss';

const Footer = () => <div className='footerContainer'><h3 className='footerText'><b>netflix</b>roulette</h3></div>


export default Footer;