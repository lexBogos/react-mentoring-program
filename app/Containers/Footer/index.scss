.footerText {
    color: rgb(246,82,97);
    font-weight: normal;
    text-align: center;
    background-color: rgb(66,66,66);
    font-size: 24px;
};

.footerContainer {
    background-color: rgb(66,66,66);
}
