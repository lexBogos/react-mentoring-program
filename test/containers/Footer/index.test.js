import React from 'react';
import Footer from '../../../app/containers/Footer/index';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Enzyme,{ shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';

Enzyme.configure({ adapter: new Adapter() });

describe('Footer component', () => {
        it('render Footer component toMatchSnapshot', () => {
            expect(shallow(<Footer/>).debug()).toMatchSnapshot();
        })
})