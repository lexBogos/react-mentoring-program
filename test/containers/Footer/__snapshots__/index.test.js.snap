// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render Footer component toMatchSnapshot 1`] = `
"<div className=\\"footerContainer\\">
  <h3 className=\\"footerText\\">
    <b>
      netflix
    </b>
    roulette
  </h3>
</div>"
`;
